tests/orm/node/test_node.py

```diff
@@ -732,13 +733,85 @@ def test_tab_completable_properties(self):
             _ = workflow.outputs.some_label
 
 
-# Clearing the DB is needed because other parts of the tests (not using
-# the fixture infrastructure) delete the User.
-def test_store_from_cache(clear_database_before_test):  # pylint: disable=unused-argument
-    """
-    Regression test for storing a Node with (nested) repository
-    content with caching.
-    """
+class TestNodeDelete(AiidaTestCase):
```

Comment:
Do these tests actually need the ``AiidaTestCase``, or could that just be removed?

Reply:
yeah, just copy/paste error, will remove it